pkg/diff/diff.go

```diff
@@ -199,7 +199,7 @@ func serverSideDiff(config, live *unstructured.Unstructured, opts ...Option) (*D
 // managedFields. All fields under this condition will be reverted with their state
 // from live. If the given predictedLive does not have the managedFields, an error
 // will be returned.
```

Comment:
Please update the doc, describing the new logic being applied and how the given `manager` parameter will affect the results.